valohai_yaml/lint.py

```diff
@@ -60,7 +64,14 @@ def lint_file(file_path: str) -> LintResult:
 def lint(yaml: YamlReadable) -> LintResult:
     lr = LintResult()
 
-    data = read_yaml(yaml)
+    try:
+        data = read_yaml(yaml)
+    except pyyaml.YAMLError as err:
+        if hasattr(err, 'problem_mark'):
+            mark = err.problem_mark
+            indent_error = f"Indentation Error at line {mark.line + 1}, column {mark.column + 1}"
+            raise ValidationError(indent_error) from err
+
```

Comment:
I think we should reraise if we didn't hit the `problem_mark` branch :)
```suggestion
        raise

```